lib/dataTableHelpers.tsx

```diff
@@ -23,3 +23,74 @@ export function truncatedTableCell(file: Entity) {
     const value = this.selector(file);
     return <ExpandableText fullText={value} truncateProps={{ lines: 4 }} />;
 }
+
+interface ISimpleColumnVisibilityDef {
+    name: string;
+    id?: string;
+    visible?: boolean;
+}
+
+export function resolveColumnVisibilityByColumnDefinition(
```

Comment:
how about `getColumnVisibilityMap`? theres not much resolving going on here, whereas in the other one you have to resolve it from the three different maps